apps/worker/src/transaction_publisher/publisher.service.ts

```diff
@@ -66,65 +66,42 @@ export class AccountUpdatePublisherService extends BaseConsumer implements OnApp
       const currentCapacityEpoch = await this.blockchainService.getCurrentCapacityEpoch();
       const providerKeys = createKeys(this.configService.getProviderAccountSeedPhrase());
       let tx: SubmittableExtrinsic<any>;
-      // TODO: Fix createSponsoredAccountWithDelegation or use siwf. TBD.
       switch (job.data.type) {
         case TransactionType.CREATE_HANDLE: {
           tx = await this.blockchainService.claimHandle(job.data.accountId, job.data.baseHandle, [
             job.data.providerId,
             job.data.payload,
           ]);
+          accountTxnHash = await this.processSingleTxn(providerKeys, tx);
+          this.logger.debug(`tx: ${tx}`);
+          this.logger.debug(`successful job: ${JSON.stringify(job, null, 2)}`);
           break;
         }
         case TransactionType.CHANGE_HANDLE: {
           tx = await this.blockchainService.changeHandle(job.data.accountId, job.data.baseHandle, [
             job.data.providerId,
             job.data.payload,
           ]);
+          accountTxnHash = await this.processSingleTxn(providerKeys, tx);
+          this.logger.debug(`tx: ${tx}`);
+          this.logger.debug(`successful job: ${JSON.stringify(job, null, 2)}`);
           break;
         }
-        case TransactionType.CREATE_ACCOUNT: {
-          // TODO: add SIWF logic & remove error.
-          throw new Error('SIWF not working yet.');
+        case TransactionType.SIWF_SIGNUP: {
+          // eslint-disable-next-line prettier/prettier
+          const txns = job.data.calls?.map((x) =>
+            this.blockchainService.api.tx(x.encodedExtrinsic),
+          );
+          const callVec = this.blockchainService.createType('Vec<Call>', txns);
+          accountTxnHash = await this.processBatchTxn(providerKeys, callVec);
+          this.logger.debug(`txns: ${txns}`);
+          this.logger.debug(`successful job: ${JSON.stringify(job, null, 2)}`);
```

Comment:
Nit: Bring outside of the switch so you only have to log once, rather than in each case.